packages/dev-console-app/src/ExtensionRow/ExtensionRow.tsx

```diff
@@ -67,7 +67,7 @@ export function ExtensionRow({
         }
       </td>
       <td className={textClass}>{extension.title}</td>
-      <td className={textClass}>{extension.type}</td>
+      <td className={textClass}>{extension.externalType}</td>
```

Comment:
In order to actually see the effect of this can you update the [test data](https://github.com/Shopify/shopify-cli-extensions/blob/main/testdata/extension.config.yml) to contain the `external_type`?